### ISSUE TITLE
A11y: Nav (old): When going to Search view the cursor is not focused on the search field

### ISSUE THREAD

**Issue by nikobearrr** [grafana/grafana#60753]
**Steps to reproduce**:
1. Open grafana
2. Click Search Dashboards
<img width="267" alt="image" src="https://user-images.githubusercontent.com/8693064/209458887-5b2cb574-34e7-4ad8-a57b-0a27fce4d2e2.png">
3. Search field is not focused

Can be reproduced by just going to https://play.grafana.org/d/000000012/grafana-play-home?orgId=1&search=open

**Actual Result**:
Nothing is focused so I need to manually click to start searching

**Expected Result**
The search field is focused so I can directly start to type. Being a "power user" I use the shortcut to open search `s o` but then I need to use my mouse to click on the search field and start typing

**Environment**:
- Grafana version: 9.3.2

**Comment by zuchka** [comment]
Thanks for creating this issue, @nikobearrr 

cna confirm that focus does not default to search field. However, you might be interested to explore the new `topnav` navigation layout:

- `-e GF_FEATURE_TOGGLES_ENABLE="topnav"`

It's a big improvement and will replace the old layout some day. It also traps focus properly when using the `s o` shortcut:

![CleanShot 2022-12-28 at 11 11 57](https://user-images.githubusercontent.com/37156449/209872792-051deaa5-b534-482f-a1f8-68063f3905f1.gif)

**Comment by JoaoSilvaGrafana** [comment]
Thanks for raising this issue @nikobearrr ! As @zuchka mentioned, we are working on a new navigation that should solve this problem of focusing on the search. This does raise a bigger issue about how hard it is for the user to use the keyboard to go from the navigation to the content. I have created an [issue](https://github.com/grafana/grafana/issues/60816) for that that we should be addressing, however we are not currently fixing any bugs on the old navigation that will be fixed with the new top nav, so will close this issue and then hopefully in a release soon you can have the top navigation turned on by default 😄
